src/features/lightspeed/api.ts

```diff
@@ -371,6 +372,11 @@ export class LightSpeedAPI {
 
       const data = await response.json();
 
+      // to remove after roleGen GA
+      if (data.role && !data.name) {
```

Comment:
This is the workaround I use to preserve the compat until the new behaviour is in prod.